README.md

```diff
@@ -6,31 +6,32 @@ Remote attestation ([RFC9334](https://datatracker.ietf.org/doc/rfc9334/)) addres
 
 This provides a much stronger assurance of trustworthiness, helping to prevent attacks that might compromise a system even if its traditional credentials remain valid, and enables authorization policies based on richer security signals.
 
-To address this need for enhanced, verifiable trustworthiness, this WG will deliver a Standards Track solution based on a post-handshake exchange for TLS that enables the attestation of one or both TLS endpoints.
-
-Such a mechanism would allow an entity to produce (fresh) Evidence or an Attestation Result about itself for another party to evaluate.
+To address this need for enhanced, verifiable trustworthiness, this WG will deliver a Standards Track solution using TLS.
+The solution will bind to the TLS handshake and enable the attestation of one or both TLS endpoints.
```

Comment:
Can we explicitly fold "binding" into (one of) the goals below? I thought that was one of the requests, but maybe I misunderstood the conclusion.

For example:
> This working group has a narrow focus: to produce a single document specifying an exchange that is performed after handshake completion and binds attestation of one or both peers to the TLS handshake.

Reply:
Thanks for the clarification.  Addressed in 9365f225fbb507dfd7d4d65d15dc1cf76b0d8c8d